README.md

```diff
@@ -76,6 +76,26 @@ or
 python3 credmaster.py --config config.json
 ```
 
+### MFASweep Module/Flag
```

Comment:
Remove this section, readme is meant for a quick usage rather than a larger walkthrough. This would be better kept in the Wiki

Reply:
yeah I can move to wiki, I've not made changes to the wiki as been keeping as a readme locally instead so pushed up with the changes to add documentation.

Reply:
All good, no need to update the wiki asap since it's still Dev branch